builders/ethereum/json-rpc/debug-trace.md

```diff
@@ -163,6 +163,61 @@ If you wish to set up your own tracing node, you can follow the [Running a Traci
           }'
         ```
 
+
+???+ function "debug_traceCall"
+
+    This method executes an eth_call within the context of the given block using the final state of the parent block as the base. Refer to [Geth's documentation](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debugtracecall){target=\_blank} for more information.
+
+    === "Parameters"
+        - `call_object` *object* the transaction object to be executed
+        - `block_hash` *string* - the block hash of the base block
+        - `tracer_config` *string* - a JSON object for configuring the tracer that contains the following field: 
+            - `tracer` *string* - sets the type of tracer. This must be set to `callTracer`, which only returns transactions and sub-calls. Otherwise, the tracer will attempt to default to the opcode logger, which is not supported at this time due to the heavy nature of the call
```

Comment:
```suggestion
            - `tracer` *string* - sets the type of tracer. Set the `tracer` field to `callTracer` to return only transactions and sub-calls. Otherwise, the tracer will default to the opcode logger, which Moonbeam does not support due to the large volume of data returned
```